ambari-server/src/main/java/org/apache/ambari/server/controller/internal/AmbariServerTProxyConfigurationHandler.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ambari.server.controller.internal;
+
+import org.apache.ambari.server.configuration.Configuration;
+import org.apache.ambari.server.events.publishers.AmbariEventPublisher;
+import org.apache.ambari.server.orm.dao.AmbariConfigurationDAO;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+/**
+ * AmbariServerTProxyConfigurationHandler is an {@link AmbariServerConfigurationHandler} implementation
+ * handing changes to the tproxy (trusted proxy) configuration
+ */
+@Singleton
+public class AmbariServerTProxyConfigurationHandler extends AmbariServerConfigurationHandler {
```

Comment:
This class doesn't seem to do anything - is it needed? Is there a generic class that could fill in here?

Reply:
It is not needed.  I originally had it override a method, but removed that and forgot to remove the empty class.  Removing...